scss/_forms.scss

```diff
@@ -288,7 +288,7 @@ input[type="checkbox"] {
   }
 }
 
-.has-error {
+.has-danger {
   @include form-control-validation(error, $brand-danger);
```

Comment:
This should also be "danger" instead of "error".
(Removing that parameter altogether is a matter for another issue/PR, #17323)